mesheryctl/internal/cli/root/app/list.go

```diff
@@ -0,0 +1,132 @@
+					UpdatedAt := fmt.Sprintf("%d-%d-%d %d:%d:%d", int(v.UpdatedAt.Month()), v.UpdatedAt.Day(), v.UpdatedAt.Year(), v.UpdatedAt.Hour(), v.UpdatedAt.Minute(), v.UpdatedAt.Second())
+					data = append(data, []string{AppID, AppName, CreatedAt, UpdatedAt})
+				}
+				utils.PrintToTableWithFooter([]string{"APP ID", "NAME", "CREATED", "UPDATED"}, data, []string{"Total", fmt.Sprintf("%d", response.TotalCount), "", ""})
+				return nil
+			}
+
+			for _, v := range response.Applications {
+				AppID := utils.TruncateID(v.ID.String())
+				var UserID string
+				if v.UserID != nil {
+					UserID = *v.UserID
+				} else {
+					UserID = "null"
+				}
+				AppName := v.Name
+				CreatedAt := fmt.Sprintf("%d-%d-%d %d:%d:%d", int(v.CreatedAt.Month()), v.CreatedAt.Day(), v.CreatedAt.Year(), v.CreatedAt.Hour(), v.CreatedAt.Minute(), v.CreatedAt.Second())
+				UpdatedAt := fmt.Sprintf("%d-%d-%d %d:%d:%d", int(v.UpdatedAt.Month()), v.UpdatedAt.Day(), v.UpdatedAt.Year(), v.UpdatedAt.Hour(), v.UpdatedAt.Minute(), v.UpdatedAt.Second())
+				data = append(data, []string{AppID, UserID, AppName, CreatedAt, UpdatedAt})
+			}
+			utils.PrintToTableWithFooter([]string{"APP ID", "USER ID", "NAME", "CREATED", "UPDATED"}, data, []string{"Total", fmt.Sprintf("%d", response.TotalCount), "", "", ""})
+
+			return nil
+		}
+
+		// Check if messhery provider is set
+		if provider == "None" {
+			for _, v := range response.Applications {
+				AppName := fmt.Sprintf("%s", strings.Trim(v.Name, filepath.Ext(v.Name)))
+				AppID := utils.TruncateID(v.ID.String())
+				CreatedAt := fmt.Sprintf("%d-%d-%d", int(v.CreatedAt.Month()), v.CreatedAt.Day(), v.CreatedAt.Year())
+				UpdatedAt := fmt.Sprintf("%d-%d-%d", int(v.UpdatedAt.Month()), v.UpdatedAt.Day(), v.UpdatedAt.Year())
+				data = append(data, []string{AppID, AppName, CreatedAt, UpdatedAt})
+			}
+			utils.PrintToTableWithFooter([]string{"APP ID", "NAME", "CREATED", "UPDATED"}, data, []string{"Total", fmt.Sprintf("%d", response.TotalCount), "", ""})
+			return nil
+		}
+		for _, v := range response.Applications {
+			AppID := utils.TruncateID(v.ID.String())
+			var UserID string
+			if v.UserID != nil {
+				UserID = *v.UserID
+			} else {
+				UserID = "null"
+			}
+			AppName := v.Name
+			CreatedAt := fmt.Sprintf("%d-%d-%d", int(v.CreatedAt.Month()), v.CreatedAt.Day(), v.CreatedAt.Year())
+			UpdatedAt := fmt.Sprintf("%d-%d-%d", int(v.UpdatedAt.Month()), v.UpdatedAt.Day(), v.UpdatedAt.Year())
+			data = append(data, []string{AppID, UserID, AppName, CreatedAt, UpdatedAt})
+		}
+		utils.PrintToTableWithFooter([]string{"APP ID", "USER ID", "NAME", "CREATED", "UPDATED"}, data, []string{"Total", fmt.Sprintf("%d", response.TotalCount), "", "", ""})
+
+		return nil
+
+	},
+}
+
+func init() {
+	listCmd.Flags().BoolVarP(&verbose, "verbose", "v", false, "Display full length user and app file identifiers")
+	listCmd.Flags().StringVarP(&token, "token", "t", "", "path to token")
+	_ = listCmd.MarkFlagRequired("token")
```

Comment:
Hey @suhaniii can you look into some latest changes in `pattern` command which help us using token without making it compulsory to pass `--token` flag.

Reply:
Thanks. I will update it.